pkg/app/pipedv1/plugin/kubernetes/deployment/plugin.go

```diff
@@ -342,9 +342,25 @@ func (p *Plugin) executeK8sRollbackStage(ctx context.Context, input *sdk.Execute
 	return sdk.StageStatusSuccess
 }
 
-// FIXME
-func (p *Plugin) DetermineVersions(context.Context, *sdk.ConfigNone, *sdk.Client, *sdk.DetermineVersionsInput) (*sdk.DetermineVersionsResponse, error) {
-	return &sdk.DetermineVersionsResponse{}, nil
+func (p *Plugin) DetermineVersions(ctx context.Context, _ *sdk.ConfigNone, _ *sdk.Client, input *sdk.DetermineVersionsInput) (*sdk.DetermineVersionsResponse, error) {
+	lp := input.Client.LogPersister()
```

Comment:
We cannot use LogPersister for anything other than ExecuteStage.
We have to use input.Logger instead.
https://github.com/pipe-cd/pipecd/blob/d50a6765488ef5fe037c1314790b8bb39123663f/pkg/plugin/sdk/client.go#L163-L166

Reply:
@Warashi 
Thanks! I forgot that 🙏 and fixed on [743bbd3](https://github.com/pipe-cd/pipecd/pull/5657/commits/743bbd3a094c4542b8f4af7c7ec056586987f93d)